java/runtime-common/src/main/java/org/ray/spi/ObjectStoreProxy.java

```diff
@@ -89,39 +82,6 @@ public void put(UniqueId id, Object obj, Object metadata) {
     store.put(id.getBytes(), Serializer.encode(obj), Serializer.encode(metadata));
   }
 
-  public <T> WaitResult<T> wait(List<RayObject<T>> waitfor, int numReturns, int timeout) {
-    List<UniqueId> ids = new ArrayList<>();
-    for (RayObject<T> obj : waitfor) {
-      ids.add(obj.getId());
-    }
-    List<byte[]> readys;
-    if (localSchedulerLink == null) {
-      readys = store.wait(getIdBytes(ids), timeout, numReturns);
-    } else {
-      readys = localSchedulerLink.wait(getIdBytes(ids), timeout, numReturns);
```

Comment:
`getIdBytes` can also be removed?

Reply:
It couldn't be removed because of some references by other methods.

Reply:
ok, I thought it wasn't used any more.